src/DonationForms/V2/DonationFormsAdminPage.php

```diff
@@ -171,8 +172,13 @@ private function preloadDonationForms()
         $queryParameters = [
             'page' => 1,
             'perPage' => 30,
+
         ];
 
+        if (isset($_GET['id']) && isset($_GET['page']) && 'give-campaigns' == isset($_GET['page'])) {
+            $queryParameters['campaignId'] = absint($_GET['id']);
```

Comment:
Campaign ID is the same as the Form ID?

Reply:
Nope! The condition is checking if we are on the campaign details page. To make it clear I created a helper function here: 56abe5317c47add063113bb0427964136ef4dd98